### ISSUE TITLE
Gamma and power settings for lambda?

### ISSUE THREAD

**Issue by NothingL** [wy1iu/sphereface#6]
Hi wy1iu,

Asoftmax is a great work.I have some questions about how to set gamma and power for lambda.
Using the settings in your prototxt
(
    base: 1000
    gamma: 0.12
    power: 1
    lambda_min: 5
)
lambda will drop very quickly to lambda_min, is it really possible to converge? 
what is the best way to setting these for different networks and different size of training data?

Thanks.

**Comment by wy1iu** [comment]
The current parameter setting can converge on CASIA with our network architectures.

I would say there is no best way, but it is generally not difficult to set these parameters. One practical trick here is to use softmax to train a model first and then use A-Softmax to finetune on it. The other trick is to save intermedia models more frequently so that you can restart your training with slightly larger lambda when your network diverges.
